scripts/load_scenarios.rb

```diff
@@ -55,7 +55,7 @@ def create_agents
   (1..NO_OF_AGENTS).each{|i|
     sh("cp -r go-agents/go-agent-#{version} go-agents/agent-#{i}")
```

Comment:
Please don't `sh("[cp|mv|rm]")` that is not portable across OSes. rake has a built in `cp`, `mv` and `rm` methods that you can use out of the box.